elasticsearch/client.go

```diff
@@ -101,6 +107,24 @@ func (cli *Client) post(ctx context.Context, index string, docType string, actio
 	return response, err
 }
 
+//CreateNewEmptyIndex is a method that creates an empty Elasticsearch index with the given indexName
+//It returns true if the index was created successfully, otherwise false
+func (cli *Client) CreateNewEmptyIndex(ctx context.Context, indexName string) (bool, error) {
+	esClient := dpelasticsearch.NewClient(cli.cfg.ElasticSearchAPIURL, cli.cfg.SignElasticsearchRequests, 5)
+	indexCreated := false
+	status, err := esClient.CreateIndex(ctx, indexName, GetSearchIndexSettings())
+	if err != nil {
+		log.Error(ctx, "error creating index. Make sure that ELASTIC_SEARCH_URL is set to http://localhost:11200", err)
```

Comment:
You've hard coded the URL value into this message but I am guessing this will be different in prod? It would be better and more flexible to add that field as `log.Data`

Reply:
Yes, good point. Done.